t/router/radixtree-sni.t

```diff
@@ -521,7 +521,82 @@ lua ssl server name: "www.test2.com"
 
 
 
-=== TEST 11: client request: aa.bb.test2.com
+=== TEST 11: client request: test.com again
```

Comment:
I think you'd better create a new test file for this PR, instead of insert one test case into the exist test cases.

Reply:
for example `multi-ssl-certs.t`

Reply:
OK

Reply:
So, can we delete this new case in the file?